bsip-0037.md

```diff
@@ -0,0 +1,64 @@
+    BSIP: 0037
+    Title: Extend created asset names
+    Author: oxarbitrage <https://github.com/oxarbitrage>
+    Status: Draft
+    Type: Protocol
+    Created: 2018-02-23
+    Discussion: https://github.com/bitshares/bitshares-core/issues/620
```

Comment:
Please create a forum thread for the discussion. Github is for a rather limited audience.